src/internal_events/aws_cloudwatch_logs_subscription_parser.rs

```diff
@@ -12,6 +12,7 @@ impl InternalEvent for AwsCloudwatchLogsSubscriptionParserError {
         error!(
             message = "Event failed to parse as a CloudWatch Logs subscription JSON message.",
             error = ?self.error,
+            error_code = "parsing_cloudwatch_logs",
```

Comment:
Seeing this now, I'm wondering if `error_code` is even useful on errors that only have one error code. Maybe `error_code` should be an optional field only added on a per-error basis if it is useful for users to be able to disambiguate. 🤔